sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -11,8 +11,9 @@ include snippets/common/legacy-maps.conf;
 
 proxy_cache_path /tmp/nginx
   levels=1:2
-  keys_zone=www_cache_zone:100m
-  inactive=120m;
+  keys_zone=NEWS_CACHE:10m
```

Comment:
Is this being lowered because we don't have enough articles to justify it?

Reply:
Nope just starting small to prevent overwhelming other parts of platform.

Reply:
Oh, I see.  So, despite having the `proxy_cache_path` the caching wasn't on and this PR turns it on?

Reply:
This is not a replacement of the caching we do on CF. 

This is more to do with making fewer queries to the upstream. So, we are only adding another layer of caching between CF and upstream.

Reply:
Yep, I'm with you now, thanks.  Is there any reason to postpone merging?